.eslintrc.js

```diff
@@ -48,7 +49,8 @@ module.exports = {
 		},
 		plugins: ['node'],
 		rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
-			'no-process-env': 0
+			'no-process-env': 0,
+			'node/no-unpublished-require': 0
```

Comment:
This rule is important to know when you use a package that is not declared on `package.json` file. Please, don't ignore it.

Reply:
Fixed. node-tests folder was not excluded from the published files in the package.